tools/query/src/utils.rs

```diff
@@ -0,0 +1,64 @@
+use anyhow::Context;
+use colored::Colorize;
+use serde_json::Value;
+
+pub fn colorize_and_print(json_str: &str) -> anyhow::Result<()> {
```

Comment:
nice work! I would checkout Diem and there is a CliTypedResult, or such, which does something similar. without colors ;)